File: FitToFit/UI/fittofit_mobile/lib/models/korisniciNovosti.dart
import 'package:json_annotation/json_annotation.dart';

part 'korisniciNovosti.g.dart';

@JsonSerializable()
class KorisniciNovosti {
  int korisniciNovostiId;
  int korisnikId;
  int novostId;
  bool isLiked;
  bool isRead;

  KorisniciNovosti(
      {required this.korisniciNovostiId,
      required this.korisnikId,
      required this.novostId,
      required this.isLiked,
      required this.isRead});

  factory KorisniciNovosti.fromJson(Map<String, dynamic> json) =>
      _$KorisniciNovostiFromJson(json);

  Map<String, dynamic> toJson() => _$KorisniciNovostiToJson(this);
}

File: FitToFit/FitToFit.Model/KorisniciNovosti.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitToFit.Model
{
    public class KorisniciNovosti
    {
        public int KorisniciNovostiId { get; set; }

        public int KorisnikId { get; set; }

        public int NovostId { get; set; }

        public bool? IsLiked { get; set; }

        public bool? IsRead { get; set; }
    }
}
